docs/content/operators/logs.md

```diff
@@ -49,15 +49,17 @@ ID** uniquely identifies a run, or execution of a bundle.
 ```console
 $ porter show whalegap
 Name: whalegap
+Bundle:
```

Comment:
Is the bundle name missing here?  Or is this how it looks when we don't know it for some reason?

Reply:
Let me change the output here to use a bundle that was installed using `--reference`. When that flag isn't used, i.e. we installed it from source, then we won't have this bundle value.

We could display `(installed from source)` instead of an empty string?

Reply:
> We could display (installed from source) instead of an empty string?

Oh maybe so, that could be a good option.  In any case, sounds like a candidate for a follow-up.